.github/workflows/e2e-tests-ci.yml

```diff
@@ -114,6 +114,9 @@ jobs:
           ln -sfn /usr/local/opt/docker-compose/bin/docker-compose ~/.docker/cli-plugins/docker-compose
           sudo ln -sf $HOME/.colima/default/docker.sock /var/run/docker.sock
       - name: ci/e2e-smoketests
+        env:
```

Comment:
I've added `cypress` to explicitly show it's using Cypress test but sounds good to me to simply run `make` knowing that it's basically smoke tests.

Reply:
Done, reverted changes here.

Reply:
Yipyip understood and agreed!
We actually only needed those empty files because the docker-compose file was committed as well, and couldn't exist without them. Since you introduced the generator, the docker-compose file is not guaranteed to exist, so we can also avoid pushing those `.env.*` files, and generate them before the docker-compose file instead.

Not blocking though, i can implement this as part of the follow-up PR.

Reply:
Awesome, thanks!

Reply:
All `.env.*` files are now generated by the generator script, and have been added to `.gitignore` https://github.com/mattermost/mattermost/pull/25040